src/MainDemo.Wpf/MainWindow.xaml.cs

```diff
@@ -114,4 +116,24 @@ private static void ModifyTheme(bool isDarkTheme)
 
     private void OnSelectedItemChanged(object sender, DependencyPropertyChangedEventArgs e)
         => MainScrollViewer.ScrollToHome();
+    private void LoadVersion()
+    {
+        try
+        {
+            string versionFilePath = Path.Combine(Directory.GetCurrentDirectory(), "version.txt");
+            if (File.Exists(versionFilePath))
+            {
+                string version = File.ReadAllText(versionFilePath).Trim();
+                VersionText.Text = $"Version: {version}";
+            }
+            else
+            {
+                VersionText.Text = "Version file not found";
+            }
+        }
+        catch (Exception ex)
+        {
+            VersionText.Text = $"Error reading version: {ex.Message}";
+        }
+    }
```

Comment:
Though the demo app isn't there to showcase best practices/design patterns, I would argue being consistent and binding a `Version` property in a MVVM way, rather than having all that logic in the code behind, is preferred.

Reply:
Thanks for comments, yoj mean that is preferable to have a class only to load the version and link it to the MVVM class? Did I understand correctly?